roles/build_openstack_packages/README.md

```diff
@@ -16,7 +16,7 @@ An Ansible role for generating custom RPMSs of OpenStack Projects using DLRN
 * `cifmw_bop_openstack_release`: (String) The default release of OpenStack project. Defaults to `master`.
 * `cifmw_bop_osp_release`: (String) The downstream openstack release name.
 * `cifmw_bop_openstack_project_path`: (String) The full path of openstack cloned project to be built.
-* `cifmw_bop_gating_repo`: (String) The path of directory to store the generated rpms.
+* `cifmw_bop_gating_repo_dest`: (String) The path of directory to store gating repo file and repo metadata.
```

Comment:
should mention the default value for the parameter?

Reply:
Done!